### PR TITLE
Bert embedding standardization

### PR DESCRIPTION
**Pull request by AlonMaor14**
Bert embedding function refactoring to adhere the new structure rules to accommodate the market place.

### CODE REVIEW

File: bert_embeddings/bert_embeddings.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Generated by nuclio.export.NuclioExporter
```
Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.py:1]: Auto generated lines should be removed

File: bert_embeddings/bert_embeddings.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import torch
+from typing import Union, List
+import json
+import pickle
+
+def init_context(context):
+    tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+    model = BertModel.from_pretrained('bert-base-uncased')
+    model.eval()
+    setattr(context.user_data, 'tokenizer', tokenizer)
```
Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.py:13]: Separate between  logical parts of the functions

File: bert_embeddings/bert_embeddings.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    setattr(context.user_data, 'model', model)
+
+def handler(context, event):
+    docs = json.loads(event.body)
+    docs = [doc.lower() for doc in docs]
+    docs = context.user_data.tokenizer.batch_encode_plus(docs, pad_to_max_length=True, return_tensors='pt')
+    with torch.no_grad():
```
Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.py:20]: Same as previous comment

File: bert_embeddings/bert_embeddings.ipynb [hunk 4]
```diff
@@ -8,85 +8,32 @@
     "This notebook presents deployment of pretrained BERT model that outputs embeddings for given textual sequences as a serverless function. Embeddings are meaningful, contextual representations of text in the form of ndarrays that are used frequently as input to various learning tasks in the field of NLP."
```
Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.ipynb:8]: <p>/# 1) function calls with a lot of parameters should be formatted as if black formatter was applied on them</p><p>/# 2) Don't mix between args and kwargs, if you name one arg, please name the rest of them</p><p><code>fn = code_to_function(</code></p><p><code>	<strong>name=</strong>items["name"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;kind=items["spec"]["kind"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;filename=items["spec"]["filename"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;description=items["description"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;categories=items["categories"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;labels=items["labels"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;requirements=items["spec"]["requirements"],</code></p><p><code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.ipynb:8]: <p>First header should be single #</p><p>Second header should be ## (i.e. the <strong>Example </strong>and <strong>Deploy  as serverless function</strong> headers)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Michaelliv) [inline bert_embeddings/bert_embeddings.ipynb:8]: <p>Please add comments that explain that this is mocked:</p><p> </p><p>before line 1:</p><p>\# Mocked Nuclio context</p><p> </p><p>before line 3:</p><p>\# Mocked conext initialization</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>
